main_test.go

```diff
@@ -185,6 +186,10 @@ func TestBuildFactsSection(t *testing.T) {
 	}
 }
 
+func TestBuildImagesSection(t *testing.T) {
+
```

Comment:
As for testing, you can basically replicate what the other section-related tests are doing (eg. `TestBuildPrimarySection`).

Create a `struct` slice which represent the test cases, taking an input `config` and then an expected `Section` struct of what you expect `buildImagesSection` to return and validate.

Reply:
Got it. Thanks for the input.

Reply:
I finished writing this test and it is passing.